spec/lib/local-links-manager/export/bad_links_url_and_status_exporter_spec.rb

```diff
@@ -0,0 +1,22 @@
+require 'rails_helper'
+require 'local-links-manager/export/bad_links_url_and_status_exporter'
+
+describe LocalLinksManager::Export::BadLinksUrlAndStatusExporter do
+  let(:exporter) { LocalLinksManager::Export::BadLinksUrlAndStatusExporter }
+
+  describe ".local_authority_bad_homepage_url_and_status_csv" do
+    it "returns the URL and status for each local authority with a non-200 homepage" do
+      allow(LocalAuthority).to receive_message_chain("where.not.distinct.pluck").and_return([["http://www.hogsmeade.gov.uk", "403"], ["http://www.godricshollow.gov.uk", "Timeout Error"], ["http://www.azkaban.gov.uk/", "Connection failed"], ["http://www.littlehangleton.gov.uk", "404"]])
```

Comment:
Stubbing `"where.not.distinct.pluck"` feels a bit brittle. Could we try replacing this with creating some links via FactoryGirl?